docs/stacks/index.md

```diff
@@ -13,30 +13,82 @@ next:
 
-A Terramate stack is:
+The Terramate CLI stack is a powerful feature of the Terramate CLI tool, allowing you to manage complex deployments with ease.
 
-* A directory inside your project.
-* Has at least one or more Terramate configuration files.
-* One of the configuration files has a `stack {}` block on it.
 
-What separates a stack from any other directory is the `stack{}` block.
-It doesn't require any attributes by default, but it can be used
-to describe stacks and orchestrate their execution.
+## What is a Stack?
+
+A stack is a collection of related resources managed as a single unit. When defining a stack, you specify all included resources, such as networks, virtual machines, and storage. 
+
+Terramate CLI simplifies resource creation and management, allowing you to focus on building and deploying the application.
+
+
+## A Terramate stack is:
+
+- A directory inside your project
+- Contains one or more Terramate configuration files
+- Includes a configuration file with a stack{} block
+
+The `stack{}` block distinguishes a stack from other directories in Terramate. By default, it doesn't require any attributes but can be used to describe stacks and orchestrate their execution.
 
 Stack configurations related to orchestration can be found [here](../orchestration/index.md).
 
-Besides orchestration the `stack` block also define attributes that are
-used to describe the `stack`.
-
-Only [Terramate Functions](../functions/index.md) are available when defining
+The `stack{}` block also defines attributes used to describe the stack. Only [Terramate Functions](../functions/index.md) are available when defining
 the `stack` block.
 
+## Why use Stacks?
+
+Stacks offer several benefits:
+
+1. Manage multiple resources as one unit, allowing you to build and deploy the entire infrastructure efficiently and quickly with a single command.
+
+2. Easily manage dependencies between resources. For example, you can specify a virtual machine that depends on a virtual network, and Terramate CLI will automatically manage the resource's creation or deletion.
+
+3. Simplify infrastructure management as code. Defining the entire infrastructure using Terraform configuration files makes version control and change management more accessible.
+
+
+## Creating a Stack:
+
+To create a stack using Terramate, follow these steps:
+
+1. Define the included resources in a Terraform configuration file. This file should contain all resources you want to manage as part of the stack.
+
+2. Use the following command to create the resources:
+
+```hcl
+terramate stack create
+```
+Running this command creates all defined resources.
+
+## Managing Stacks
+
+After creating a stack, you can use Terramate CLI to manage it. 
+
+- To **update** the stack, use:
+```hcl
+terramate stack update
+```
+
+- To **delete** the entire stack you can use the command:
+```hcl
+terramate stack delete
+```
```

Comment:
`terramate stack` does not exist actually. I am wondering where this is coming from and how you came with this? So far we have `terramate create` for creating new stacks.